concreate/module.py

```diff
@@ -63,13 +63,14 @@ def get_dependencies(descriptor, base_dir):
       descriptor - image descriptor
       base_dir - root directory for dependencies
     """
-    logger.debug("Retrieving dependencies for %s" % (descriptor['name']))
-    if 'dependencies' not in descriptor:
-        logger.debug("No dependencies specified in descriptor")
+    logger.debug("Retrieving module repositories for '%s'" % (descriptor['name']))
+
+    if not descriptor.module_repositories:
```

Comment:
this can be else branch again - its more pythonic

Reply:
This will be left as-is.